### DISCUSSION TITLE
Welcome to magni Discussions!

### DISCUSSION THREAD

**Discussion by ctrlw** [ctrlw/magni#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BisbeeSteve2** [comment]
BisbeeSteve2 is my username. I am old and memory not the greatest. Last September 2024, I assembled the magnifier with a R Pi Zero 2W and the V.3 camera and it worked fine. Before I could build a support for it, I tore everything down to evacuate because of a wildfire.  I finally got around to putting it back together and setting up a stand for the device.  It didn't fire up like it was supposed to so I reflashed the sd with RPi OS 32 bit Lite and followed the instructions from magni/README.md at master - ctrl/magni  .
I ran rasp-config and set System options-> Boot/Autologin -> Console Autologin
Hit Finish and reboot
In the terminal I ran the following 👍 sudo apt update && sudo apt -y upgrade
That ran fine.
Next I ran the following:
sudo apt install -y python3-picamera2 python3-gpiozero python3-evdev python3-open
  Responds with
Reading package lists...  Done
Building dependency tree...  Done
Reading state information...  Done
E: Unable to locate package python3-open

I checked Copilot on my Win11 machine and it tells me python3-open is not a valid package and wasn't in Sept 2024 either.  As I write this ,it occurs to me that perhaps the full list of commands was not printed when I captured the page so I will go ahead and post this and check out the rest of the line if it exists.
However if someone wants to muse upon my shortcomings, feel free.

**Reply by ctrlw** [reply]
Hi, as you've figured out already, the line you pasted was incomplete and the actual package is `python3-opencv`, i.e. you were missing the last 2 letters. All the best with the rest of the setup :-)

**Reply by BisbeeSteve2** [reply]
Rest of  setup is still not working.  See the screenshot please.

**Comment by BisbeeSteve2** [comment]
you for checking.  I originally set up the pi zero 2w last September and it instantly worked! Had to evacuate for a  forest fire and finally got it set up again but I'm having a paucity of luck.. Had some internet issues Apparently and some packages had issues. I finally reflashed the SD and started over from scratch. The camera still hasn't started so I'm not out of the woods yet. Once my computer room cools down today, I will  take another stab at it. If you don' t mind, I may ask you   more questions if I can't figure them out on my own . Aging is really a bummer. Memory is my worst issue with debugging. I have to check code about 5 times more often than just a few years ago.

Writing this on a kindle and some things just disappeared like the thank you that was in the first line.

Anyway, thank you for doing this project. I have had cataract surgery in both eyes, and then had secondary cataracts which are now corrected. All this drug out over~5 years and I spent a lot of time reading with a Merlin ultra hd that was donated to the library. When I found the article in MagPi I was overjoyed and knew I just had to do it.

Thank you,
Steven Lueck

**Comment by BisbeeSteve2** [comment]
Adding the cv took care of that issue.  Now I have found a couple of bugs that are probably just typos or remnants of previous versions.  I'll attach a picture of my screen.  I could just start changing out capital letters in nano but I thought you might like to see it first.  Please let me know if I have done something egregious. I am illiterate in github and even worse in Reddit.  Thank you in advance.

![2025-08-11_magni_issues](https://github.com/user-attachments/assets/15ff1d7b-d652-42ac-bf57-f095000737b9)
